dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ManualRegistry.java

```diff
@@ -50,9 +50,9 @@ public ManualRegistry(Registry registry) {
 
     @Override
     public void updateAPIToolsWithBuildInformation(List<Tool> apiTools) {
-        for (Tool tool : apiTools) {
-            tool.setRegistry(registry);
-        }
+//        for (Tool tool : apiTools) {
```

Comment:
Accidental or incomplete commenting?

Reply:
That is too remove, I just forgot to fully remove it after I commented it out.